beps/0001-notifications-system/README.md

```diff
@@ -178,6 +178,25 @@ Let's walk through the process of sending of a notification to a user:
 
    7b. Rather than rendering the notification directly, the notifications frontend plugin leaves that to an extension installed by the frontend part of the backend plugin that sent the notification. This allows for a more customized notification experience which can also contain custom actions and data display. This step is not in scope for this proposal.
 
+### Integration with external systems
+
+The `notifications-backend` is not meant to expose the API outside of the Backstage, only other Backstage backend services are expected to connect through the NotificationsService.
```

Comment:
Hmm, it is, it has a REST API. It just happens that other Backstage plugins should use the service instead, as a more convenient TypeScript interface. It's implemented here: https://github.com/backstage/backstage/blob/0b120df27bffb774acdd09464ce47ce071f1e74e/plugins/notifications-backend/src/service/router.ts#L278